cmd/minishift/cmd/root.go

```diff
@@ -109,12 +109,15 @@ var RootCmd = &cobra.Command{
 		// creating all directories for minishift run
 		createMinishiftDirs(state.InstanceDirs)
 
+		// Ensure the global viper config file exists.
+		ensureGlobalConfigFileExists(constants.GlobalConfigFile)
+
```

Comment:
-1 the content of `ensureGlobalConfigFileExists` and `ensureConfigFileExists` is same. We do not need to have two functions when they exactly do the same thing.